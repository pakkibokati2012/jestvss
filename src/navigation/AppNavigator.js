import { createAppContainer, createSwitchNavigator } from 'react-navigation';
import AuthLoadingScreen from '../screens/authentication/AuthLoadingScreen';
import AuthNavigator from './AuthNavigator';
import MainNavigator from './MainNavigator';

export default createAppContainer(
  createSwitchNavigator(
    {
      AuthLoading: AuthLoadingScreen,
      Auth: AuthNavigator,
      Main: MainNavigator
    },
    {
      initialRouteName: 'AuthLoading'
    }
  )
);
