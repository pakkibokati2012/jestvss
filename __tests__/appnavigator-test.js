/**
 * @format
 */

import 'react-native';
import React from 'react';
import AppNavigator from '../src/navigation/AppNavigator';
import renderer from 'react-test-renderer';

test('AppNavigator Snapshot Testing', () => {
  const snap = renderer.create(<AppNavigator />).toJSON();
  expect(snap).toMatchSnapshot();
});
