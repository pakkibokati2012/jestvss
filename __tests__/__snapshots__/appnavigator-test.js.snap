// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNavigator Snapshot 1`] = `null`;

exports[`AppNavigator Snapshot Testing 1`] = `null`;
